indico/modules/designer/util.py

```diff
@@ -10,13 +10,27 @@
 
 from indico.core.db import db
 from indico.modules.designer.models.templates import DesignerTemplate
+from indico.modules.designer.pdf import PIXELS_CM
 from indico.modules.designer.placeholders import GROUP_TITLES
 from indico.modules.events.models.events import Event
 from indico.modules.events.registration.models.forms import RegistrationForm
 from indico.util.date_time import now_utc
 from indico.util.placeholders import get_placeholders
 
 
+FORMAT_MAP_PORTRAIT = {
+        'A0': (84.1, 118.9),
```

Comment:
Too much indentation

Reply:
Done, I didn't notice it while copy-pasting.